src/components/common/footer/Footer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '../../../../public/styles/Footer.scss';
+import facebook from '../../../../public/images/facebook-logo-button.png';
+import twitter from '../../../../public/images/twitter-logo-button.png';
+
+const Footer = () => (
+  <footer className="footer">
+    <div className="footer-content">
+      <div className="d-flex justify-content-around bd-highlight text-center">
+        <div className="bd-highlight"><p className="text-muted m-0">Copyright 2018, Author’s Haven</p></div>
+        <div className="bd-highlight">
+          <div>
+            <img className="mr-3" src={facebook} alt="facebook logo" />
```

Comment:
I suggest you nest the img tag in an anchor tag. The facebook and twitter log should be clickable.